object/object.go

```diff
@@ -447,17 +453,27 @@ func (o *Object) OwnerID() *user.ID {
 		copy(id[:], b)
 	}
 
-	return &id
+	return id
 }
 
 // SetOwnerID sets identifier of the object owner.
 //
 // See also [Object.OwnerID].
+// Deprecated: use [Object.SetOwner] accepting value instead.
 func (o *Object) SetOwnerID(v *user.ID) {
+	if v == nil {
+		_ = v[:] // just to cause panic
```

Comment:
How is it different from panicking two lines below?

Reply:
sure, some forgotten stuff from me, dropped